app/lib/common/utils/utils.dart

```diff
@@ -36,8 +36,10 @@ final idMatrixRegexp = RegExp(
   r'matrix:roomid/(?<id>[^?]+)(\?via=(?<server_name>[^&]+))?(&via=(?<server_name2>[^&]+))?(&via=(?<server_name3>[^&]+))?',
 );
 
-final urlValidatorRegexp = RegExp(
```

Comment:
and the regexp grows, yet now tests are nowhere to be found ...

how are we sure we don't break stuff, if we don't have tests for this?